source/extensions/filters/http/gcp_authn/gcp_authn_filter.h

```diff
@@ -17,6 +17,9 @@ namespace GcpAuthn {
 
 inline constexpr absl::string_view FilterName = "envoy.filters.http.gcp_authn";
 inline constexpr absl::string_view AudienceKey = "audience_key";
+inline const std::string& authorizationHeaderKey() {
```

Comment:
You can make it `constexpr absl::string_view` just like the others. I think it should work.